ui/components/AutomationDetail.tsx

```diff
@@ -179,6 +180,24 @@ function AutomationDetail({
       },
       visible: true,
     },
+    {
+      name: "Violations",
+      path: `${path}/violations`,
+      component: () => {
+        return (
+          <PolicyViolationsList
+            req={{
+              application: name,
+              clusterName,
+              namespace,
+              kind: type,
+            }}
+            sourcePath="kustomization"
+          />
+        );
+      },
+      visible: !customTabs?.find((c) => c.name === "Violations"),
```

Comment:
Because we want to prevent an dependencies for now as we won't use the same Violation component in the WGE so separate things is a good way to do so, and yes there will be a follow up PR to export and make use of it in WGE once it's ready. 

taking in consideration the validation check over the CanaryStatus depends on customTabs .